services/token_get_info.proto

```diff
@@ -182,6 +182,16 @@ message TokenInfo {
      * The custom fees to be assessed during a CryptoTransfer that transfers units of this token
      */
     repeated CustomFee custom_fees = 23;
+
+    /**
+     * The Key which can pause and unpause the Token.
+     */
+    Key pauseKey = 24;
+
+    /**
+     * Specifies whether the token is paused or not.
+     */
+    bool paused = 25;
```

Comment:
Would it make sense to make this a `TokenPauseStatus` and have a `NotApplicable` value for tokens with no pause key, similar to freeze and kyc?

Reply:
Makes Sense, will update the proto.